crawlmanager/crawl.py

```diff
@@ -11,6 +11,9 @@
 from starlette.exceptions import HTTPException
 from ujson import dumps as ujson_dumps
 
+import logging
```

Comment:
is there any reason why we are jacking uvicorns logger rather than using our own?

Also we should keep all imports sorted like below and move the logger variable below `__all__``

```suggestion
from __future__ import annotations

import logging
import uuid
from asyncio import AbstractEventLoop, gather as aio_gather, get_event_loop
from functools import partial
from typing import Dict, List, Optional, Union
from urllib.parse import urlsplit

from aiohttp import AsyncResolver, ClientSession, TCPConnector
from aioredis import Redis
from starlette.exceptions import HTTPException
from ujson import dumps as ujson_dumps

from .schema import CrawlInfo, CreateCrawlRequest, StartCrawlRequest
from .utils import env, init_redis

__all__ = ['Crawl', 'CrawlManager']

logger = logging.getLogger('uvicorn')
```